packages/block-library/src/comments/index.js

```diff
@@ -17,4 +18,12 @@ export const settings = {
 	icon,
 	edit,
 	save,
+	variations: [
+		{
+			name: 'default',
+			isDefault: true,
+			innerBlocks: TEMPLATE,
+			scope: [ 'inserter' ],
+		},
+	],
```

Comment:
Do we still need the `variations`? I _think_ we tried this approach to [work around the issue with detecting inner blocks](https://github.com/WordPress/gutenberg/pull/40522#issuecomment-1108780293), but it looks like we'd be avoiding this issue now anyway if we go with the `legacy` attribute as discussed during our call 🙂

Reply:
Good point @ockham. I'll remove the `variations` attribute. 👍